BDKSwiftExampleWallet/View/SendView.swift

```diff
@@ -6,8 +6,111 @@
+    // Sync
+    @Published var lastSyncTime: Date? = nil
+    @Published var walletSyncState: WalletSyncState = .notStarted
+    
+    // Transactions
+    @Published var transactionDetails: [TransactionDetails] = []
+    
+    func getBalance() {
+        do {
+            let balance = try BDKService.shared.getBalance()
+            self.balanceTotal = balance.total
+            self.balanceSpendable = balance.spendable
+            self.balanceImmature = balance.immature
+            self.balanceTrustedPending = balance.trustedPending
+            self.balanceUntrustedPending = balance.untrustedPending
+            self.balanceConfirmed = balance.confirmed
+        } catch let error as WalletError {
+            print("getBalance - Wallet Error: \(error.localizedDescription)")
+        } catch {
+            print("getBalance - Undefined Error: \(error.localizedDescription)")
+        }
+    }
+    
+    func getTransactions() {
+        do {
+            let transactionDetails = try BDKService.shared.getTransactions()
+            self.transactionDetails = transactionDetails
+        } catch {
+            print("getTransactions - none: \(error.localizedDescription)")
+        }
+    }
+    
+    func sync() async {
+        DispatchQueue.main.async {
+            self.walletSyncState = .syncing
+        }
+        Task {
+            do {
+                try await BDKService.shared.sync()
+                DispatchQueue.main.async {
+                    self.walletSyncState = .synced
+                    self.lastSyncTime = Date()
+                    self.getBalance()
+                    self.getTransactions()
+                }
+            } catch {
+                DispatchQueue.main.async {
+                    self.walletSyncState = .error(error)
+                }
+            }
+        }
+    }
+}
+
+extension SendViewModel {
+    
+    enum WalletSyncState: CustomStringConvertible, Equatable {
+        case error(Error)
+        case notStarted
+        case synced
+        case syncing
+        
+        var description: String {
+            switch self {
+            case .error(let error):
+                return "Error Syncing: \(error.localizedDescription)"
+            case .notStarted:
+                return "Not Started"
+            case .synced:
+                return "Synced"
+            case .syncing:
+                return "Syncing"
+            }
+        }
+        
+        static func ==(lhs: WalletSyncState, rhs: WalletSyncState) -> Bool {
+            switch (lhs, rhs) {
+            case (.error(let error1), .error(let error2)):
+                return error1.localizedDescription == error2.localizedDescription
+            case (.notStarted, .notStarted):
+                return true
+            case (.synced, .synced):
+                return true
+            case (.syncing, .syncing):
+                return true
+            default:
+                return false
+            }
+        }
+    }
+}
```

Comment:
Lets remove this since we won't need any sync related functionality on this screen